src/deploy/calculate-address/calculate-dpos.ts

```diff
@@ -21,10 +21,14 @@ const deploy = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
       validatorContract: calculateAddress(deployer, nonce++),
     };
   }
+
+  // console.debug('Nonce calculation for deployments...');
+  // console.table(generalRoninConf[network.name]);
```

Comment:
```suggestion
```

Reply:
Saving this line for the sake of debugging next time. I have rewritten these lines hundreds of times.